pkgs/pkgs-lib/formats.nix

```
@@ -1125,4 +1125,34 @@ optionalAttrs allowAliases aliases
     else
       throw "pkgs.formats.xml: Unknown format: ${format}";
 
+  plist =
+    {
+      escape ? true,
+    }:
+    {
+      type =
+        let
+          valueType =
+            nullOr (oneOf [
+              bool
+              int
+              float
+              str
+              path
+              (attrsOf valueType)
+              (listOf valueType)
+            ])
+            // {
+              description = "Proper list value";
```

I'm sure you meant to write `Property list`, but most people are probably so used to calling it plist that it would be more quickly recognizable to just use the shorthand here.

```suggestion
              description = "Plist value";
```